### PR TITLE
Selenium: Add changes to selenium tests according to new client

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Add some changes to selenium tests from 'intelligencecommand' according  removing sshd installer from default stacks
__Related selenium tests: MacrosCommandsEditorTest_, AutocompleteCommandsEditorTest_

* Add some change to the selenium test _'CheckIntelligenceCommandFromToolbarTest'_  to successful passing both platform che6 (docker and ocp)

### What issues does this PR fix or reference?
#7713

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java [hunk 1]
```diff
@@ -62,6 +63,12 @@ public void launchClonedWepAppTest() throws Exception {
     projectExplorer.waitItem(PROJECT_NAME);
     commandsToolbar.clickWithHoldAndLaunchCommandFromList(PROJECT_NAME + ": build and run");
     consoles.waitExpectedTextIntoConsole(" Server startup in");
+
+    String previewUrl = consoles.getPreviewUrl();
+    if (previewUrl.contains("route")) {
+      WaitUtils.sleepQuietly(10);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: Could you, please, add comment here with explanation what such a big timeout is for?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java [hunk 2]
```diff
@@ -78,14 +85,26 @@ public void checkButtonsOnToolbar() {
     projectExplorer.waitProjectExplorer();
     String currentWindow = seleniumWebDriver.getWindowHandle();
     commandsToolbar.clickExecStopBtn();
+
+    String previewUrl = consoles.getPreviewUrl();
+    String expectedText =
+        previewUrl.contains("route")
+            ? "Application is not available"
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: Could you, please, comment here why we depend on the preview url in such a strange manner?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java (hunk 1 above)
Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: It need when the test is running on che6-ocp platform

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: I see. Please, add comment into the code to make it clear for other developers.

**Comment by dmytro-ndp** [comment]
IMHO sooner or later we will need to find clearer way to express the difference between the tests which are executing on the _che6_ based on **docker** and _che6_ based on **ocp**.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java (hunk 2 above)
Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: Also, it needs when the test is running on che6-ocp platform

**Comment by artaleks9** [comment]
I, perhaps, agree, but right now we need to go on the such solution

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java (hunk 2 above)
Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: I see. Please, also add comment into the code to make it clear for other developers.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java (hunk 1 above)
Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java]: Have done

**Review by SkorikSergey — APPROVED** [review]

**Pull request merged**